### DISCUSSION TITLE
Gave a Heart?

### DISCUSSION THREAD

**Discussion by MMendelZ** [ScratchAddons/ScratchAddons#5143] [Ideas]
is it possible to make a block in the sensing category that that checks weather if use your heart?

**Comment by BroJac5246** [comment]
They're workarounds, but nothing that Scratch Addons could or would implement.

Check out https://scratch.mit.edu/discuss/topic/620643/, or search for a tutorial.

**Reply by WorldLanguages** [reply]
Yeah, this is out of scope for SA.
